app/views/content/international-qualified-teacher-status/_gain-iqts-and-automatic-qts.md

```diff
@@ -0,0 +1,3 @@
+At the end of your training, your iQTS provider and school will assess you to check you meet iQTS [Teachers’ Standards](https://www.gov.uk/government/publications/international-qualified-teacher-status-teachers-standards). 
```

Comment:
```suggestion
At the end of your training, your iQTS provider and school will assess you to check you meet [iQTS Teachers’ Standards](https://www.gov.uk/government/publications/international-qualified-teacher-status-teachers-standards). 
```
need to include iQTS in link for consistency/accessibility - and to be clear that this is different from QTS standards